### PR TITLE
Add PyTorch and MXNet example with various readers

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
Builds 40755

File: docs/examples/mxnet/mxnet-various-readers.ipynb [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+    "- TFRecordReader\n",
+    "\n",
+    "For details on how to use them please see other [examples](..)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lets start from defining some global constants"
```
Comment (Kh4L) [inline docs/examples/mxnet/mxnet-various-readers.ipynb]: I would say:
```suggestion
    "Let's start with defining some global constants"
```

File: docs/examples/pytorch/pytorch-various-readers.ipynb [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+    "- TFRecordReader\n",
+    "\n",
+    "For details on how to use them please see other [examples](..)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lets start from defining some global constants"
```
Comment (Kh4L) [inline docs/examples/pytorch/pytorch-various-readers.ipynb]: Same here

**Comment by JanuszL** [comment]
Builds 40861

**Comment by JanuszL** [comment]
Builds 591844

**Pull request merged**